x-pack/plugins/integration_assistant/server/util/samples.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import * as yaml from 'js-yaml';
+import { dump } from 'js-yaml';
```

Comment:
Maybe we should use `safeDump` here?

Reply:
I think it makes sense to use `safeDump`, not sure it should be change in the pr however. The idea was to keep the logic and just remove the `*` from import by specifying the function used